beacon-chain/main.go

```diff
@@ -8,11 +8,19 @@ import (
 	"github.com/prysmaticlabs/prysm/beacon-chain/utils"
 	"github.com/prysmaticlabs/prysm/shared/cmd"
 	"github.com/prysmaticlabs/prysm/shared/debug"
-	log "github.com/sirupsen/logrus"
+	logger "github.com/sirupsen/logrus"
```

Comment:
maybe not necessary to rename to logger since we're mostly using the return value of `logrus.WithField`

Reply:
it'll also avoid a naming collision with the Logger interface